Engine/Communication/sender/grsim_command_sender.py

```diff
@@ -49,3 +50,12 @@ def _create_protobuf_packet(packets_frame) -> grSim_Packet.grSim_Packet:
             grsim_command.kickspeedz = 0
 
         return grsim_packet
+
+    @staticmethod
+    def translate_kick_force(kick_force: KickForce) -> int:
+        kick_translation = {KickForce.NONE: 0,
+                            KickForce.LOW: 1,
+                            KickForce.MEDIUM: 3,
+                            KickForce.HIGH: 5}
```

Comment:
Should be much lower I think. Please test it in simulation

Reply:
It looked fine. But we can tweak them